src/community/propose-a-content-change-using-github/index.md.njk

```diff
@@ -0,0 +1,93 @@
+---
+title: Propose a content change using GitHub
+description: Find out how to suggest a change to content in the GOV.UK Design System using GitHub
+section: Community
+layout: layout-pane.njk
+order: 2
+---
+
+The GOV.UK Design System team uses a service called GitHub to manage content in the GOV.UK Design System.
+
+This guide explains how to propose a change to the Design System's content. You will need a GitHub account to do this.
+
+If you do not have one already, you can [create a GitHub account for free](https://github.com/).
+
+{% from "inset-text/macro.njk" import govukInsetText %}
+{{ govukInsetText({
+  text: "Rest assured that it's impossible for you to break the Design System by proposing changes. The GOV.UK Design System team reviews all changes before publishing."
```

Comment:
I hear this tested well which is great! But the wording feels a bit odd to me. Would 'You cannot' get across the same thing?

```suggestion
  text: "You cannot break the Design System by proposing changes. The GOV.UK Design System team reviews all changes before publishing."
```

Reply:
I totally take your point but my sense was that people liked the slightly more human language used here so I'm going to leave this as is if your comment isn't blocking.

Reply:
The only thing we might want to be careful with is making statements like 'impossible' as it might encourage malicious people who see it as a challenge. 🕵

Reply:
Do you think we need to make a change then?

Reply:
I think the risk exists either way, so let's leave it...